src/vs/base/browser/ui/hover/hover.css

```diff
@@ -131,3 +131,9 @@
 	border-bottom: 1px solid transparent;
 	text-underline-position: under;
 }
+
+/** Spans in markdown hovers need a margin-bottom to avoid looking cramped: https://github.com/microsoft/vscode/issues/101496 **/
+.monaco-hover .markdown-hover .hover-contents span {
+	margin-bottom: 4px;
```

Comment:
Maybe this should use `em` so it is updated automatically if the font size changes